### DISCUSSION TITLE
Install wordpress in github pages

### DISCUSSION THREAD

**Discussion by proarafat** [community/community#22015] [Other Feature Feedback, Questions, & Ideas]
<p>Can we install GitHub using GitHub pages? Or we can add mySQL and database in the wordpress in github?</p>

**Comment by jfullstackdev** [comment]
<aside class="quote no-group" data-username="proarafat" data-post="1" data-topic="242406">
<div class="title">
<div class="quote-controls"></div>
 proarafat:</div>
<blockquote>
<p>Can we install GitHub using GitHub pages?</p>
</blockquote>
</aside>
<p>what do you mean by that ? GitHub Pages is an added service so that a repo can be simply turned to a website,</p>
<aside class="quote no-group" data-username="proarafat" data-post="1" data-topic="242406">
<div class="title">
<div class="quote-controls"></div>
 proarafat:</div>
<blockquote>
<p>Or we can add mySQL and database in the wordpress in github?</p>
</blockquote>
</aside>
<p>GitHub Page is browser-side only, just HTML, CSS and JavaScript directly, but you can have a workaround like providing the link on the GitHub Page, but redirecting it to where you can run server-side script</p>

**Comment by David-Lor** [comment]
<p>Something you could try (if it fits your needs) is to host (e.g. on your localhost) a Wordpress. then use a plugin that exports the whole blog as static html, and upload it to a Github repository with Pages enabled. However in this case it may be easier to just use something like Jekyll or Hugo.</p>
<p>Theorically you could host a Wordpress and MySQL using Github Actions, but only for testing purposes, without persistent storage and for a limited time.</p>

**Comment by proarafat** [comment]
<p>I say that is we can use a database system in our codes and run the website using Github pages?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>GitHub Pages only runs directly browser-side, if there is a technology, language or framework that can communicate ( all browser-side ) to a server that has the access to a database, then your fine,</p>
<p>unfortunately, as far as I know, those full-pledge database are server-side</p>

**Comment by CharlesRakesh** [comment]
https://github.com/CharlesRakesh/Elementor-Pro-Form/issues/1
